src/components/transition/Transition.jsx

```diff
@@ -258,7 +258,7 @@ function BaseTransition({
     }
 
     if (supportsTransitions() && isFillModeBackwards(fillMode)) {
-      animator.animate(container, rules.from);
+      animator.apply(container, rules.from);
```

Comment:
It's about "prefilling" the element before actual animation (related to filling modes) so from the beginning, it should be a different method than `animate`.